README.adoc

```diff
@@ -178,16 +153,15 @@ Create the `InventoryManager` class in the
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=add_fallback]
 ----
 
-The `@Fallback` annotation dictates which method to call when the automated retries are unsuccessful. In this
+The `@Fallback` annotation dictates a method to call when encounters a failed execution. In this
 example, use the `fallbackForGet` method.
 
 The `fallbackForGet()` method, which is the designated fallback method for the original
-`get()` method, checks to see if the system's properties exist in the inventory. If the properties
-variable is null, the system properties entry is not found in the inventory.
-Thus, the method prints out a JSON warning
-message in the browser. Otherwise, this method returns the cached property values.
+`get()` method, checks to see if the system's properties exist in the inventory.
+If the system properties entry is not found in the inventory, the method prints out a warning
+message in the browser. Otherwise, this method returns the cached property values from the inventory.
 
-You have now successfully set up the application to be fault tolerant.
+You have now successfully set up your fault tolerant microservices.
```

Comment:
"fault-tolerant" needs a hyphen here because it's a compound adjective in front of a noun.

Reply:
We try to avoid the perfect tense ("have + verb"), but we're a little more relaxed about this since the guides are more conversational. Even so, what about..."You successfully set up your fault-tolerant microservices."?

Reply:
How about changing it to, "You have now successfully set up your microservice to have fault tolerance capability." or "You have now successfully set up your microservice to have fallback capability."

Reply:
The issue is the perfect tense "have," which is why I suggested "You successfully set up your fault-tolerant microservices."

However, if we need to use the perfect tense, we can stick with how it is now (with the exception of adding the hyphen, of course): "You have now successfully set up your fault-tolerant microservices."

Reply:
How about, "You successfully set up your microservice to have fault tolerance capability."?

Reply:
Yes, that works. :)